tests/testthat/test-utils.R

```diff
@@ -115,6 +115,8 @@ test_that("buffering a bbox does not change its CRS", {
   expect_equal(crs_actual, crs_expected)
 })
 
+#' @srrstatsTODO {G2.4, G2.4c} Explicit conversion to character with
+#'   `as.character()` used to test the output type of `buffer()`.
```

Comment:
I am not sure if test code like this falls within the scope of G2.4. Maybe all test code should fall under G5.

Reply:
Not sure about tests, but we should check all uses oof`sf::st_geometry_type()`, which returns a factor - so it should involve G2.4e:
```
G2.4e explicit conversion from factor via `as...()` functions
```
Here, for instance we use `as.character()`

Reply:
Updated with the last one.